journal-of-physical-therapy-science.csl

```diff
@@ -0,0 +1,151 @@
+  </macro>
+  <macro name="publisher">
+    <text variable="publisher-place" suffix=": "/>
+    <text variable="publisher" suffix=", "/>
+  </macro>
+  <macro name="container-title">
+    <choose>
+      <if type="chapter paper-conference" match="any">
+         <text term="in" text-case="capitalize-first" suffix=": "/>
+         <text variable="container-title"/>
+      </if>
+      <else-if type="article article-magazine article-newspaper article-journal" match="any">
+         <text variable="container-title" form="short" strip-periods="true"/>
+      </else-if>
+      <else>
+          <text variable="container-title"/>
+      </else>
+    </choose>     
+  </macro>
+  <macro name="title">
+    <choose>
+      <if type="bill book graphic legal_case legislation motion_picture report song" match="any">
+        <text variable="title" suffix=", "/>
+      </if>
+      <else>
+        <text variable="title" suffix=". "/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="edition">
+    <choose>
+      <if is-numeric="edition">
+        <group delimiter=" ">
+          <number variable="edition" form="ordinal"/>
+          <text term="edition" form="long" suffix="."/>
+        </group>
+      </if>
+      <else>
+        <text variable="edition" suffix="."/>
+      </else>
+    </choose>
+  </macro>
+  <citation collapse="citation-number">
+    <sort>
+      <key variable="citation-number"/>
+    </sort>
+    <layout suffix=")" delimiter=", " vertical-align="sup">
+      <text variable="citation-number"/>
+    </layout>
+  </citation>
+  <bibliography et-al-min="4" et-al-use-first="3" delimiter-precedes-et-al="always" second-field-align="flush" entry-spacing="0">
+    <layout>
+      <group  vertical-align="sup">
+        <text variable="citation-number" suffix=") "/>
+      </group>
+      <choose>
+        <if type="webpage">
+          <text macro="title"/>
+          <text variable="URL"/>
+          <text macro="accessed" prefix=" (" suffix=")"/> 
+        </if>
+        <else>
+          <text macro="author"/>
+          <text macro="title"/>
+          <choose>
+            <if type="bill book graphic legal_case legislation motion_picture report song" match="any">
+              <group delimiter=" ">
+                <text macro="edition"/>
+                <text macro="publisher"/>
+              </group>
+            </if>
+            <else-if type="chapter paper-conference" match="any">
+              <text macro="editor"/>
+              <text macro="container-title" suffix=". "/>
+              <group delimiter=" ">
+                <text macro="edition" suffix="."/>
+                <text macro="publisher"/>
+              </group>
+            </else-if>
+            <else>
+              <text macro="container-title" suffix=", "/>
+            </else>
+          </choose>
+          <date variable="issued">
+            <date-part name="year" suffix=", "/>
+          </date>
+          <text variable="volume" suffix=": "/>
```

Comment:
right, what I meant with "Books and Chapter can have volumes too" is that you might have those in the Zotero data for a book item, and then they'd show up in citations, which, as you note, doesn't correspond to the guide. 
Thanks for fixing this, let us know when you're done.